azure-pipelines.yml

```diff
@@ -25,22 +46,43 @@ stages:
         submodules: false
         coverage: codecov
         toxdeps: tox-pypi-filter
+        envs:
+          - linux: codestyle
+            name: style_check
+            pytest: false
+            libraries: {}
+          - linux: py39
 
-        libraries:
-          apt:
-            - libopenjp2-7
-            - graphviz
-          brew:
-            - openjpeg
-
+  - stage: SecondPhaseTests
+    displayName: Stage 2 Tests
+    dependsOn: FirstPhaseTests
+    jobs:
+    - template: run-tox-env.yml@OpenAstronomy
+      parameters:
+        default_python: '3.8'
+        submodules: false
+        coverage: codecov
+        toxdeps: tox-pypi-filter
         envs:
           - macos: py37
           - windows: py38
-          - linux: py38
           - linux: build_docs
             posargs: " "
             pytest: false
-          - linux: codestyle
-            name: style_check
-            pytest: false
-            libraries: {}
+
+  # On branches which aren't main, and not Pull Requests, build the wheels but only upload them on tags
```

Comment:
Can we leave the wheel building out for now too?

Reply:
A universal wheel for a pure python package is simple enough. It might change if we have any compiled code.